packages/commonwealth/client/scripts/views/pages/overview/index.tsx

```diff
@@ -76,72 +77,75 @@ const OverviewPage = () => {
   return !topicSummaryRows.length ? (
     <PageLoading />
   ) : (
-    <div className="OverviewPage">
-      <div className="header-row">
-        <div className="header-row-left">
-          <CWText type="h3" fontWeight="semiBold">
-            Overview
-          </CWText>
-          <CWButton
-            className="latest-button"
-            buttonType="primary"
-            buttonHeight="sm"
-            label="Latest Threads"
-            iconLeft="home"
-            onClick={() => {
-              navigate('/discussions');
-            }}
-          />
+    <>
+      <GatingGrowl />
```

Comment:
instead of creating Fragment Wrapper, you can put GatingGrowl in  `OverviewPage` div as a first child I guess. CW growl component is positioned fixed so it does not matter where you put it. This way we will avoid editing the whole component